### PR TITLE
Vcf fusions

### PR DESCRIPTION
**Pull request by rannick**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnafusion/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/rnafusion _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by rannick** [comment]
To add:
- HGNC id
-  transcript annotations
- transcript versions
- exon number

File: bin/megafusion.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+import argparse
+import csv
+import logging
+import os
+import sys
+from collections import Counter
+from pathlib import Path
+import pandas as pd
+import ast
+import re
```
Comment (jemten) [inline bin/megafusion.py:12]: Are all these used? Can't find that `csv`, `os`, `Counter` and `re` is used.

**Review by jemten — COMMENTED** [review]
Some first comments on the python script :) 
Regarding the INFO description in the VCF header it was a bit of guessing from my side as I haven't seen the test data

File: bin/megafusion.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+##INFO=<ID=GENEB,Number=.,Type=String,Description="Gene B">\n\
+##INFO=<ID=ORIENTATION,Number=.,Type=String,Description="Strand1 and strand2 directions">\n\
+##INFO=<ID=FOUND_DB,Number=.,Type=String,Description="Databases in which the fusion has been found">\n\
+##INFO=<ID=ARRIBA,Number=.,Type=String,Description="Found by arriba">\n\
+##INFO=<ID=FUSIONCATCHER,Number=.,Type=String,Description="Found by fusioncatcher">\n\
+##INFO=<ID=PIZZLY,Number=.,Type=String,Description="Found by pizzly">\n\
+##INFO=<ID=SQUID,Number=.,Type=String,Description="Found by squid">\n\
+##INFO=<ID=STARFUSION,Number=.,Type=String,Description="Found by starfusion">\n\
+##INFO=<ID=TOOL_HITS,Number=.,Type=String,Description="Number of tools that found the fusion">\n\
+##INFO=<ID=SCORE,Number=.,Type=String,Description="Score from fusionreport">\n\
```
[the remark this answers is not in the document]
Comment (rannick) [reply bin/megafusion.py]: It is a float!

**Review by jemten — APPROVED** [review]
👍

**Pull request merged**